docs/advanced_topics/api/v2/extra_usage.rst

```diff
@@ -0,0 +1,86 @@
+=====================================
+Extra information on exposing the API
+=====================================
+
+Images
+------
+
+If you have in your model an image you can expose the image by using the ImageRenditionField.
+:class:`wagtail.images.api.fields.ImageRenditionField`. The ImageRenditionField will return url, width, height.
+
+..code-block:: python
```

Comment:
It would be good to add a line with the extra imports required for this example as not everyone will know where to look for imports.
eg.

```
from wagtail.images.api.fields import ImageRenditionField
```